react/w-books/src/app/screens/Login/components/LoginForm/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ChangeEvent, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useForm } from 'react-hook-form';
+import { Redirect } from 'react-router-dom';
+
+import Form from '../../../../components/Form';
+import FormRow from '../../../../components/FormRow';
+import Button from '../../../../components/Button';
+import PATHS from '../../../../components/Routes/paths';
+import Loading from '../../../../components/Loading';
+import ErrorMessage from '../../../../components/ErrorMessage';
+
+import styles from './styles.module.scss';
+
+import { useLazyRequest } from '../../../../../hooks/useRequest';
+import { LOGIN, postUser } from '../../../../../services/userService';
+
+
+interface UserData {
+  email?: string;
+  password?: string;
+}
+
+export default function LoginForm() {
+  const { t } = useTranslation();
+  const { errors, register, handleSubmit } = useForm<UserData>();
+
+  const [userData, loading, error, request] = useLazyRequest({
+    request: (data: UserData) => postUser(LOGIN, data),
+  })
+
+  const login = (formData: UserData) => {
+    if (formData) {
+      request(formData)
+    }
+  }
+
+  return (
+    userData ?
+    <Redirect to={PATHS.home} /> :
+    <Form className={styles.loginForm} handleSubmit={handleSubmit(login)}>
+      <FormRow
+        labelName={t('LoginForm:UserFormEmail')}
+        className={styles.formRow}
+        inputId="email"
+        inputType="email"
+        inputRef={
+          register({
+            required: t('FormValidation:Required'),
+            pattern: {
+              value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
+              message: t('FormValidation:Format')
+            }
+          })
+        }
+        errorMessage={errors.email?.message}
+      />
+      <FormRow
+        labelName={t('LoginForm:UserFormPassword')}
+        className={styles.formRow}
+        inputId="password"
+        inputType="password"
+        inputRef={
+          register({
+            required: t('FormValidation:Required'),
+          })
+        }
+        errorMessage={errors.password?.message}
+      />
+
+      {error && (<ErrorMessage>{t(`LoginForm:${error.problem}`)}</ErrorMessage>)}
+
+      {
+        loading ? (<div className={styles.loading}> <Loading isGreen /> </div>)
+          :<Button isFilled isSubmit>{t('Common:LoginButton')}</Button>
+      }
```

Comment:
This seems weird... do you have eslint correctly configured?

Reply:
yes, i run it and does nothing in this line maybe need something like prettier or my config its bad?